api-samples/tabCapture/manifest.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Tab Capture Example",
+  "description": "Capture a tab and play in a <video> element in a separate tab.",
```

Just noticed this, sorry!

```suggestion
  "description": "Capture a tab and play in a <video> element in a separate window.",
```